test/test_other.py

```diff
@@ -2369,6 +2369,11 @@ def test_sdl2_ttf(self):
     self.emcc(test_file('browser/test_sdl2_ttf.c'), args=['-sUSE_SDL=2', '-sUSE_SDL_TTF=2'], output_filename='a.out.js')
     self.emcc(test_file('browser/test_sdl2_ttf.c'), args=['--use-port=sdl2', '--use-port=sdl2_ttf'], output_filename='a.out.js')
 
+  def test_contrib_ports(self):
+    # Verify that contrib ports can be used (using the only contrib port available ATM, but can be replaced
+    # with a different contrib port when there is another one
+    self.emcc(test_file('other/test_contrib_ports.cpp'), ['--use-port=contrib.glfw3'], output_filename='a.out.js')
```

Comment:
I think you can drop `output_filename='a.out.js'`

Reply:
ok. I copy/pasted the million other tests around that do thit.

Reply:
Oh, sorry, actually this might be needed, sorry I didn't see this was a call to `self.emcc`.   I guess if the test passes without it that is fine.

Reply:
it looks like it works without it on my local machine. I suppose we will find out if it fails on CI...